backend/db/ddl-21-sessions.jinja.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS public.sessions{{optional_suffix}} CASCADE;
```

Comment:
I can't remember, are we currently using this table?

I think you said maybe not? I saw you were doing "`next_api_call_group_id()`" to get session IDs.